diff --git a/src/solidarity.ts b/src/solidarity.ts
index c5284c3..abb3a30 100644
--- a/src/solidarity.ts
+++ b/src/solidarity.ts
@@ -23,7 +23,7 @@ import fs from "fs";
 
 const PATTERN = /((?:(?:white|black)[_-]*list)|slave|master)/gi;
 const SUMMARY = fs.readFileSync("./static/HELP.md", "utf8");
-const CHECK_NAME = "Inclusive Language Check";
+const CHECK_NAME = "Inclusive Language";
 
 export enum Conclusion {
   SUCCESS = "success",
diff --git a/test.txt b/test.txt
deleted file mode 100644
index c5284c3..abb3a30 100644
--- a/test.txt
+++ /dev/null
@@ -1 +0,0 @@
-master